### PR TITLE
Multi-module Maven Project using Java Modules

### PR DESCRIPTION
**Pull request by alejandrogervasio**

### CODE REVIEW

File: maven-java-11/pom.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (KevinGilmore) [inline maven-java-11/pom.xml]: Need to add this module to the root pom.xml, and then in this pom.xml, need to aggregate the multimodule-maven-project.

Comment (alejandrogervasio) [reply maven-java-11/pom.xml]: Hey @KevinGilmore No worries. I'll do that today and post accordingly.

File: maven-java-11/multimodule-maven-project/pom.xml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.baeldung.multimodule-maven-project</groupId>
```
Comment (KevinGilmore) [inline maven-java-11/multimodule-maven-project/pom.xml]: All of the other modules declare a parent. Do you think this one should declare maven-java-11 as its parent? Just wondering if it was omitted intentionally or not.

File: maven-java-11/multimodule-maven-project/userdaomodule/pom.xml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.baeldung.multimodule-maven-project</groupId>
+        <artifactId>multimodule-maven-project</artifactId>
+        <version>1.0</version>
+    </parent>
+    <groupId>com.baeldung.userdaomodule</groupId>
+    <artifactId>userdaomodule</artifactId>
+    <packaging>jar</packaging>
```
Comment (KevinGilmore) [inline maven-java-11/multimodule-maven-project/userdaomodule/pom.xml]: Need to add a <version> tag

File: maven-java-11/multimodule-maven-project/mainppmodule/pom.xml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.baeldung.multimodule-maven-project</groupId>
+        <artifactId>multimodule-maven-project</artifactId>
+        <version>1.0</version>
+    </parent>
+    <groupId>com.baeldung.mainappmodule</groupId>
+    <artifactId>mainappmodule</artifactId>
+    <packaging>jar</packaging>
```
Comment (KevinGilmore) [inline maven-java-11/multimodule-maven-project/mainppmodule/pom.xml]: do we need to add a <version> tag?

File: maven-java-11/multimodule-maven-project/userdaomodule/pom.xml (hunk 3 above)
Comment (alejandrogervasio) [reply maven-java-11/multimodule-maven-project/userdaomodule/pom.xml]: Yes. We need to. Good catch! I'll add it ASAP.

File: maven-java-11/multimodule-maven-project/mainppmodule/pom.xml (hunk 4 above)
Comment (alejandrogervasio) [reply maven-java-11/multimodule-maven-project/mainppmodule/pom.xml]: We need the version tag in this one. I'll add it as well.

File: maven-java-11/multimodule-maven-project/pom.xml (hunk 2 above)
Comment (alejandrogervasio) [reply maven-java-11/multimodule-maven-project/pom.xml]: Yes. I think this one should declare maven-java-11 as its parent. I'll add the tags.

**Pull request merged**